.github/workflows/update-deb-package-snapshots.yml

```diff
@@ -0,0 +1,26 @@
+name: update-snapshots
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
Might be better to schedule this slightly off-peak?  Help Github avoid computation cascades at midnight?

Reply:
Good idea. Going with UTC 8:30am (EST 4:30 and 1:30 PST as of now).